murenn/dtcwt/nn.py

```diff
@@ -179,13 +191,13 @@ def __init__(self, J_phi):
             J=1,
             level1="near_sym_b",
             skip_hps=True,
-            padding_mode="zeros",
         )
+        self.relu = torch.nn.ReLU()
 
 
     def forward(self, x):
         for j in range(self.J_phi):
             x, _ = self.phi(x)
             # Normalize the coefficients
```

Comment:
remove this comment. you are not normalizing the coefficients anymore. you are directly applying a decimation `::2`